app/parsers/bulkrax/xml_parser.rb

```diff
@@ -78,16 +79,19 @@ def file_paths
     # Otherwise return all xml files in the given folder
     def metadata_paths
       @metadata_paths ||=
-        if file? && MIME::Types.type_for(import_file_path).include?('application/xml')
+        if file? && good_file_type?(import_file_path)
           [import_file_path]
         else
           file_paths.select do |f|
-            MIME::Types.type_for(f).include?('application/xml') &&
-              f.include?("import_#{importerexporter.id}")
+            good_file_type?(f) && f.include?("import_#{importerexporter.id}")
           end
         end
     end
 
+    def good_file_type?(path)
+      File.extname(path) || ::Marcel::MimeType.for(path).include?('application/xml')
```

Comment:
What is `File.extname(path)` doing here?

Reply:
It's supposed to check the extension, except I didn't add it... fixed this in my latest commit.